browser/BUILD.gn

```diff
@@ -106,6 +110,8 @@ source_set("browser_process") {
     "//brave/components/brave_ads/browser",
     "//brave/components/brave_component_updater/browser",
     "//brave/components/brave_drm",
+    "//brave/components/brave_prochlo",
```

Comment:
why are these prochlo deps here? Shouldn't they be a deps of //brave/components/p3a?

Reply:
https://github.com/brave/brave-core/pull/3242/files#diff-6014e7308fa8986675992b1869b4fefaR19

Reply:
fixed